option.go

```diff
@@ -119,6 +119,18 @@ func EqualFuncs(b bool) Option {
 	}}
 }
 
+// ZeroFields copies values and sets the specified fields to their zero value.
```

Comment:
These docs need to say that it is a transform, then it can say what the transform does. Otherwise this description won't make much sense to the reader.

Probably it should also say or at least imply that T has to be a struct. Maybe rename `T` to `Struct`?

Reply:
Maybe like `ZeroFields transforms a value of struct type T. It makes a copy of its input and sets the specified fields to their zero values.`

Reply:
Come to think of it, it should probably also say what the practical effect of all this is, which is to cause those fields to be ignored. That's the whole point of this lol.

```
// ZeroFields transforms a value of struct type T. It makes a copy of its input
// and sets the specified fields to their zero values.
//
// This effectively causes the given fields to be ignored for comparison.
//
// See also Transform.
```

Reply:
Agreed!